src/frontend/apps/impress/src/features/docs/doc-editor/components/DocEditorSkeleton.tsx

```diff
@@ -0,0 +1,156 @@
+import { css, keyframes } from 'styled-components';
+
+import { Box } from '@/components';
+import { tokens, useCunninghamTheme } from '@/cunningham';
+import { useResponsiveStore } from '@/stores';
+
+const colors = tokens.themes.default.theme.colors;
+
+const shimmer = keyframes`
+  0% {
+    background-position: -1000px 0;
+  }
+  100% {
+    background-position: 1000px 0;
+  }
+`;
+
+const SkeletonLine = ({
+  width = '100%',
+  height = '16px',
+  margin = '0',
+}: {
+  width?: string;
+  height?: string;
+  margin?: string;
+}) => {
```

Comment:
Can we extend the interface with `BoxType` ?
Like that we will benefit from all the props of Box.

Here an example of implementation: https://github.com/suitenumerique/docs/blob/main/src/frontend/apps/impress/src/components/Overlayer.tsx